.ci/azure-pipelines/build_macos.bash

```diff
@@ -48,3 +48,6 @@ cmake -G Ninja -DCMAKE_TOOLCHAIN_FILE=$MUMBLE_ENVIRONMENT_TOOLCHAIN -DIce_HOME="
 
 cmake --build .
 
+
+# Delete the environment in order to ensure that we have enough space for the packaging step
+rm -rf "$MUMBLE_ENVIRONMENT_PATH$"
```

Comment:
This should be done only for pull requests, otherwise we delete the build environment on our own builder.